honeycomb-beeline.gemspec

```diff
@@ -52,5 +52,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rubocop-performance", "< 1.3.0"
   spec.add_development_dependency "simplecov"
   spec.add_development_dependency "simplecov-console"
-  spec.add_development_dependency "webmock"
+  webmock_version = RUBY_VERSION < "2.3.0" ? "< 3.15.0" : nil
+  spec.add_development_dependency "webmock", webmock_version
```

Comment:
It only makes my brain itch a little bit. I might offer the more verbose rendition of this because ternaries make my eyes cross sometimes:

```ruby
if RUBY_VERSION < "2.3.0"
  spec.add_development_dependency "webmock", "< 3.15.0"
else
  spec.add_development_dependency "webmock"
end
```

But my brain itches are not blockers.